src/app/pages/account/groups/group-list/group-list.component.spec.ts

```diff
@@ -115,10 +115,10 @@ describe('GroupListComponent', () => {
       expect(snapshots).toEqual([]);
     });
 
-    const table = await loader.getHarness(IxTableHarness);
-    const text = await table.getCellTextByIndex();
+    spectator.detectChanges();
 
-    expect(text).toEqual([['Can not retrieve response']]);
+    const emptyTitle = spectator.query('.empty-title');
```

Comment:
Would be good to have a harness here, so that `.empty-title` class is hidden away.